pkg/ddc/alluxio/ufs_internal.go

```diff
@@ -372,8 +372,8 @@ func (e *AlluxioEngine) genUFSMountOptions(m datav1alpha1.Mount, SharedOptions m
 
 	// initialize alluxio mount options
 	mOptions := map[string]string{}
-	if len(SharedOptions) > 0 {
-		mOptions = SharedOptions
+	for k, v := range SharedOptions {
```

Comment:
Could you also add unit test for the fix? thanks very much!

Reply:
I have added unit test: generating UFS mount options multiple times via functions `genUFSMountOptions` in `AlluxioEngine`